Update storage-use-azcopy-blobs-synchronize.md (#126968)
Azure/azure-content: articles/storage/common/storage-use-azcopy-blobs-synchronize.md

azcopy sync 'https://mysourceaccount.blob.core.windows.net/<container-name>/myDirectory?sv=2018-03-28&ss=bfqt&srt=sco&sp=rwdlacup&se=2019-07-04T05:30:08Z&st=2019-07-03T21:30:08Z&spr=https&sig=CAfhgnc9gdGktvB=ska7bAiqIddM845yiyFwdMH481QA8%3D' 'https://mydestinationaccount.blob.core.windows.net/mycontainer/myDirectory' --recursive
```

> ## Update a container with changes in another container using SAS-token

Reviewer: ```suggestion
## Update a container with changes in another container by using a SAS token
```

Author: Changes made as requested at line 111.

The first container that appears in this command is the source. The second one is the destination.

> if you provide SAS-token make sure you use SAS-token corresponding to source and destination storage-account, while using **azcopy sync**

Reviewer: ```suggestion
If you provide a SAS token make sure that you use a SAS token which corresponds to source and destination storage account, while using **azcopy sync**
```

Author: Changes made as requested at line 115

> If you provide a SAS token make sure that you use a SAS token which corresponds to source and destination storage account, while using **azcopy sync**

Reviewer 2: ```suggestion
If you provide a SAS token, make sure that you use a SAS token which corresponds to source and destination storage account, while using **azcopy sync**.
```

**Syntax**

`azcopy sync 'https://<source-storage-account-name>.blob.core.windows.net/<container-name>/<SAS-token>' 'https://<destination-storage-account-name>.blob.core.windows.net/<container-name>/<destination-SAS-token>' --recursive`

**Example**

```azcopy
azcopy sync 'https://mysourceaccount.blob.core.windows.net/mycontainer?sv=2018-03-28&ss=bfqt&srt=sco&sp=rwdlacup&se=2019-07-04T05:30:08Z&st=2019-07-03T21:30:08Z&spr=https&sig=CAfhgnc9gdGktvB=ska7bAiqIddM845yiyFwdMH481QA8%3D' 'https://mydestinationaccount.blob.core.windows.net/mycontainer/?sv=2018-03-28&ss=qrdu&dst=mco&kp=rwdlapuc&se=2019-07-04T05:30:08Z&st=2019-07-03T21:31:10Z&spr=https&sig=CAfhgnc9gdGktvB=yzx9CAjMKeeN678yiyFwdNU092JC8%6D' --recursive
```

> > [!TIP]

Reviewer: ```suggestion

```

Author: Provided recommendation below for "https://learn.microsoft.com/en-us/answers/questions/1162473/how-to-find-the-size-of-a-folder-in-azure-data-lak" for Tip section

> > If you are copying larger size of data, you can use :[Azure Storage Explorer](https://azure.microsoft.com/en-us/products/storage/storage-explorer).

Reviewer: ```suggestion

```

Reviewer: Storage Explorer is not what the storage team recommends for larger data sets. We have defined guidance here - https://learn.microsoft.com/en-us/azure/storage/common/storage-choose-data-transfer-solution?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json.

Author: I completely agree with you, as we do not recommend using Storage Explorer for large data transfers. My intention here is to provide customers who are transferring large datasets using AzCopy or AzCopy sync with a helpful tip. Specifically, I suggest using the 'Folder Statistics' option in Storage Explorer to compare dataset file sizes especially when customers are copying terabytes of data. This can serve as a useful reference point during large-scale transfers. I was referring a page - https://learn.microsoft.com/en-us/answers/questions/1162473/how-to-find-the-size-of-a-folder-in-azure-data-lak  

Please suggest so that i can modify the "tips section accordingly

> > If you are copying larger size of data, you can use :[Azure Storage Explorer](https://azure.microsoft.com/en-us/products/storage/storage-explorer).

Reviewer 2: ```suggestion
> If you are copying larger size of data, you can use :[Azure Storage Explorer](https://azure.microsoft.com/products/storage/storage-explorer).
```
fixing localization issue

> 

> > Verify size details that you have copied On the top bar menu choose **More** -> **Folder Statistics** will help you get the details of the Directory including the size in bytes.

Reviewer: ```suggestion

```

Author: Provided recommendation below for "https://learn.microsoft.com/en-us/answers/questions/1162473/how-to-find-the-size-of-a-folder-in-azure-data-lak" for Tip section

> > Verify size details that you have copied On the top bar menu choose **More** -> **Folder Statistics** will help you get the details of the Directory including the size in bytes.

Reviewer 2: ```suggestion
> Verify size details that you have copied. On the top bar menu, choose **More** -> **Folder Statistics** to help you get the details of the Directory including the size in bytes.
```
cleaning up grammar

|Specify how strictly MD5 hashes should be validated when downloading.|**--check-md5**=\[NoCheck\|LogOnly\|FailIfDifferent\|FailIfDifferentOrMissing\]|
|Exclude files based on a pattern.|**--exclude-path**|
|Specify how detailed you want your sync-related log entries to be.|**--log-level**=\[WARNING\|ERROR\|INFO\|NONE\]|

> |Specify how to copy vhd file.|**--blob-type=BlockBlob --include-pattern "*.vhd"** or **--blob-type=BlockBlob** |

Reviewer: ```suggestion
|Specify how to copy a vhd file.|**--blob-type=BlockBlob --include-pattern "*.vhd"** or **--blob-type=BlockBlob** |
```

Author: Changes made as requested at line 142.
